index.html

```diff
@@ -307,6 +311,9 @@ <h3>
       </li>
       <li>If |requests| is empty, [=exception/throw=] a {{TypeError}}.
       </li>
+      <li>If |requests| does not contain [=JSON type=] values,
+        [=exception/throw=] a {{TypeError}}.
+      </li>
```

Comment:
This is not right here... you need to: 

1. [=List/for each=] |request| of  |requests|:
1.1. [=serialize a JavaScript value to a JSON string|Serialize=] |request| a JSON string. Rethrow any exceptions.

Reply:
@timcappalli, we also would need this on the way out, when `readonly attribute object data;` is set... though there might not be enough spec text in the current spec to add this right now. However, it would be good to add something.

Reply:
> readonly attribute object data

The benefit of also doing this on the way back is that we could make this integrate well with `fetch()` ... arguably, we could also this in a separate PR ...

> This is not right here... you need to:

Yeah, we'd still want to do this, I think.

Reply:
No. This is because if the wallet returns some bogus value that is not a JSON *object* (e.g., a string, or a number, will TypeError at this point), then that can't map to `.data` (because it's required to be an *object*). Thus you need to JSON parse the response _before_ it is stored in `.data`... then the developer calling `.toJSON()` walks both the DigitalCredential instance _and_ the `. data`. structure. 

Make sense?